e2e/scripts/st_arrow_line_chart_add_rows_special.py

```diff
@@ -0,0 +1,18 @@
+import time
+
+import pandas as pd
+import streamlit as st
+
+
+current_time = pd.to_datetime("08:00:00 2021-01-01", utc=True)
+simulation_step = pd.Timedelta(seconds=10)
+
+df1 = pd.DataFrame(data=[[current_time, 1]], columns=["t", "y"]).set_index("t")
+line_chart = st._arrow_line_chart(df1, use_container_width=True)
```

Comment:
Plz add a comment for the `_arrow_line_chart` function call. (Presumably we want to force arrow serialization, and not worry about whether Streamlit is configured for legacy dataframe serialization?)